tsl/src/init.c

```diff
@@ -144,6 +145,9 @@ CrossModuleFunctions tsl_cm_functions = {
 	.policies_alter = policies_alter,
 	.policies_show = policies_show,
 
+	/* Vectorized queries */
+	.is_agg_capable_node = apply_vectorized_agg_optimization,
+
```

Comment:
Since this is not only about the general possibility of aggregation in this node, but also actually alters the path dependent on the particular query, probably it should have a different name, maybe something like `.push_down_aggregation`, not sure. All in all I think this is a useful API that can also be used by CNS.

Reply:
Naming changed to `push_down_aggregation` as suggested